src/index.html

```diff
@@ -28,17 +28,29 @@ <h3>Friends:</h3>
       <h3><span class="user-friends-js"></span></h3>
     </div>
     <div class="tile div4">
+        <h3 class="heading">Step Data</h3>
       <h3>Daily Step Goal: <span class="user-daily-step-goal-js"></span></h3>
-    </div>
-    <div class="tile div5">
       <h3>Other People's Average Step Goal:</h3>
       <h3><span class="step-goal-comparison-js"></span></h3>
     </div>
+    <div class="tile div5">
```

Comment:
Populating the Sleep Data Tile by interpolating all the info through our scripts.js file.